app/src/main/kotlin/rasel/lunar/launcher/settings/childs/Apps.kt

```diff
@@ -111,6 +119,19 @@ internal class Apps : BottomSheetDialogFragment() {
 
         return binding.root
     }
+    @SuppressLint("RtlHardcoded")
+    override fun onCreateDialog(savedInstanceState: Bundle?): Dialog {
```

Comment:
`onCreateDialog` is called before `onCreateView`

Reply:
Thanks, I'm not really familiar with Android or Kotlin in General, but I was just hacking through it to get desired result on my end.

Reply:
I am also doing the same thing, along with the help of web. :smiley: